markup_fmt/src/parser.rs

```diff
@@ -353,14 +353,27 @@ impl<'s> Parser<'s> {
                 !c.is_ascii_whitespace() && !matches!(c, '"' | '\'' | '=' | '<' | '>' | '`')
             }
 
-            let Some((start, _)) = self.chars.next_if(|(_, c)| is_unquoted_attr_value_char(*c))
-            else {
-                return Err(self.emit_error(SyntaxErrorKind::ExpectAttrValue));
+            let start = match self.chars.peek() {
+                Some((i, c)) if is_unquoted_attr_value_char(*c) => { *i },
+                _ => return Err(self.emit_error(SyntaxErrorKind::ExpectAttrValue))
             };
-            let mut end = start;
 
-            while let Some((i, _)) = self.chars.next_if(|(_, c)| is_unquoted_attr_value_char(*c)) {
-                end = i;
+            let mut end = start;
+            loop {
+                match self.chars.peek() {
+                    Some((_, '{')) => {
```

Comment:
Here's an edge case: `attr={` isn't interpolation. Also, we should check language here.